plugins/default-validator/default-validator.go

```diff
@@ -121,6 +128,37 @@ func (v *DefaultValidator) validateOCIHooks(req *api.ValidateContainerAdjustment
 	return fmt.Errorf("%w: %s attempted restricted OCI hook injection", ErrValidation, offender)
 }
 
+func (v *DefaultValidator) validateNamespaces(req *api.ValidateContainerAdjustmentRequest) error {
+	if req.Adjust == nil {
+		return nil
+	}
+
+	if !v.cfg.RejectNamespaces {
+		return nil
+	}
+
+	owners, claimed := req.Owners.NamespaceOwners(req.Container.Id)
+	if !claimed {
+		return nil
+	}
+
+	offenders := ""
+	sep := ""
+
+	if len(owners) > 1 {
+		offenders = "plugins "
+	} else {
+		offenders = "plugin "
+	}
```

Comment:
nit: maybe simply `offenders := "plugin(s)"` :)

Reply:
@djdongjin Updated PR like that.